pytket/extensions/cutensornet/tnstate/ttn_gate.py

```diff
@@ -309,57 +450,75 @@ def _apply_2q_gate(self, q0: Qubit, q1: Qubit, gate: Op) -> TTNxGate:
             this_fidelity = 1.0 - svd_info.discarded_weight
             self.fidelity *= this_fidelity
 
-            # Contract V to the parent node of the bond

```

Comment:
This code has been moved to the method `_contract_decomp_bond_tensor_into_ttn` so that it can be used both by the `chi` and the `truncation_fidelity` approaches. The code itself has not changed.